src/test/java/com/keeper/homepage/global/util/redis/RedisUtilTest.java

```diff
@@ -0,0 +1,62 @@
+package com.keeper.homepage.global.util.redis;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.keeper.homepage.IntegrationTest;
+import java.util.Optional;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class RedisUtilTest extends IntegrationTest {
+
+  @Test
+  @DisplayName("직렬화, 역직렬화가 잘 동작해야 한다.")
+  void should_serializeAndDeserializeIsOk() {
+    TestDto testDto = new TestDto(1, "AA");
+    redisUtil.setDataExpire("0", testDto, 10000);
+
+    Optional<TestDto> getTestDto = redisUtil.getData("0", TestDto.class);
+    assertThat(getTestDto).isNotEmpty();
+    assertThat(getTestDto.get().a).isEqualTo(1);
+    assertThat(getTestDto.get().b).isEqualTo("AA");
```

Comment:
```suggestion
    assertThat(getTestDto.get()).usingRecursiveComparison().isEqualTo(testDto);
```
사소하지만 내부값만 비교하고자할 때는 이렇게도 비교할 수 있습니다 :)

Reply:
어메이징